features/guidance/anticipate-lanes.feature

```diff
@@ -582,9 +582,9 @@ Feature: Turn Lane Guidance
 
     @anticipate
     Scenario: No Lanes for Roundabouts, see #2626
-        Given the profile file "car" extended with
+        Given the profile file "car" initialized with
         """
-        properties.left_hand_driving = true
+        profile.left_hand_driving = true
```

Comment:
@TheMarex requested that properties be grouped under profile.properties. see discussion at https://github.com/Project-OSRM/osrm-backend/pull/4072#issuecomment-309457857